libs/sdk-core/src/lnurl/pay.rs

```diff
@@ -159,6 +160,7 @@ pub(crate) mod model {
     #[derive(Serialize, Deserialize, Debug)]
     pub struct LnUrlPaySuccessData {
         pub payment_hash: String,
+        pub payment: Payment,
```

Comment:
Why do we keep `payment_hash` given the new field ?

Reply:
> Why do we keep `payment_hash` given the new field ?

Good point, the `payment.id` can be used